main.go

```diff
@@ -1,48 +1,132 @@
+var lorem = `<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Qui potest igitur habitare in beata vita summi mali metus? Praeclare Laelius, et recte sofñw, illudque vere: O Publi, o gurges, Galloni! es homo miser, inquit. Nam si propter voluptatem, quae est ista laus, quae possit e macello peti? Quae diligentissime contra Aristonem dicuntur a Chryippo. In quibus doctissimi illi veteres inesse quiddam caeleste et divinum putaverunt. Bork Quaeque de virtutibus dicta sunt, quem ad modum eae semper voluptatibus inhaererent, eadem de amicitia dicenda sunt. Summus dolor plures dies manere non potest? </p>
 
 <p>Duo Reges: constructio interrete. Quid ergo aliud intellegetur nisi uti ne quae pars naturae neglegatur? At Zeno eum non beatum modo, sed etiam divitem dicere ausus est. Si longus, levis; Polemoni et iam ante Aristoteli ea prima visa sunt, quae paulo ante dixi. Hoc est non modo cor non habere, sed ne palatum quidem. Quid, si etiam iucunda memoria est praeteritorum malorum? </p>
 
 <p>Id enim volumus, id contendimus, ut officii fructus sit ipsum officium. Quo modo? Expressa vero in iis aetatibus, quae iam confirmatae sunt. Quodsi vultum tibi, si incessum fingeres, quo gravior viderere, non esses tui similis; Esse enim, nisi eris, non potes. </p>`
 
-	lines := strings.Split(lorem, "\n")
+func print_frequency_map(frequency_map map[int][]string) {
+	for key, element := range frequency_map {
+		fmt.Println(key)
+		for _, word := range element {
+			fmt.Println(word)
+		}
+	}
+}
+
+type WordCount struct {
+	Word  string `json:"Word"`
+	Count int    `json:"Count"`
+}
+
+func word_json(lorem io.Reader) string {
+	// url := "loripsum.net/api"
+
+	newScanner := bufio.NewScanner(lorem)
 
-	// fmt.Println(lines)
-	// fmt.Println(len(lines))
+	newScanner.Split(bufio.ScanWords)
 
 	counts := map[string]int{}
 
-	for _, line := range lines {
-		words := strings.Split(line, " ")
-		for _, word := range words {
-			counts[word]++
-		}
+	for newScanner.Scan() {
+		counts[newScanner.Text()]++
+	}
+
+	var wordCountArray []WordCount
+	for key, value := range counts {
+		wordCountArray = append(wordCountArray, WordCount{key, value})
 	}
 
-	j, err := json.Marshal(counts)
+	sort.Slice(wordCountArray, func(p, q int) bool {
+		if wordCountArray[p].Count != wordCountArray[q].Count {
+			return wordCountArray[p].Count > wordCountArray[q].Count
+		}
+		return wordCountArray[p].Word > wordCountArray[q].Word
+	})
+
+	j, err := json.Marshal(wordCountArray)
 	if err != nil {
 		panic(err)
```

Comment:
Now that this logic exists in its own function outside `main()`, I'd say it's worth making this function return `(string, error)` so it can bubble up any errors it encounters to its caller which might be able to handle them better than `panic`-ing.